tests/integration/docker_images/popular-containers-images.bats

```diff
@@ -0,0 +1,405 @@
+	$DOCKER_EXE run --rm -i oraclelinux bash -c 'for NUM in `seq 1 1 10`; do touch $NUM-file.txt; done; ls -l /1-file.txt'
+}
+
+@test "check os version in an alpine container" {
+	$DOCKER_EXE run --rm alpine cat /etc/alpine-release
+}
+
+@test "show the help menu in an arangodb container" {
+	skip "this image is not working (see https://github.com/01org/cc-oci-runtime/issues/320)"
+	$DOCKER_EXE run --rm -e ARANGO_ROOT_PASSWORD=secretword -e ARANGO_NO_AUTH=1 -p 8529:8529 arangodb/arangodb foxx-manager help | grep "update"
+}
+
+@test "start solr server" {
+	$DOCKER_EXE run --rm -i solr timeout 10 solr -h test | grep "Solr"
+}
+
+@test "check hostname in a clearlinux container" {
+	$DOCKER_EXE run --rm -i clearlinux bash -c "hostname"
+}
+
+@test "show the os version inside a buildpack deps container" {
+	$DOCKER_EXE run --rm -i buildpack-deps cat /etc/os-release | grep "PRETTY_NAME"
+}
+
+@test "display message in a julia container" {
+	$DOCKER_EXE run --rm -i julia bash -c "julia -e 'println(\"this is a test\")'"
+}
+
+@test "start wordpress container" {
+	skip "this image is not working (see https://github.com/01org/cc-oci-runtime/issues/318)"
+	if $DOCKER_EXE run -i wordpress perl -e 'print "test\n"' | grep "LANG"; then false; else true; fi
+}
+
+@test "display the cqlsh version in a cassandra container" {
+	$DOCKER_EXE run --rm -i cassandra cqlsh --version
+}
+
+@test "cat a file in a centos container" {
+	$DOCKER_EXE run --rm -i centos bash -c "echo "Test" > testfile.txt | cat testfile.txt | grep "Test""
+}
+
+@test "run a couchbase container" {
+	skip "this image is not working (see https://github.com/01org/cc-oci-runtime/issues/318)"
+	$DOCKER_EXE run -i couchbase couchbase-cli rebalance --help | grep "rebalance"
+}
+
+@test "display options in a chronograf container" {
+	$DOCKER_EXE run --rm -i chronograf -sample-config | grep "configuration"
+}
+
+@test "run an instance in a traefik container" {
+	skip "this image is not working (see https://github.com/01org/cc-oci-runtime/issues/222)"
+	if $DOCKER_EXE run -i traefik traefik --version | grep "EXEC spawning"; then false; else true; fi
+}
+
+@test "run a consul container" {
+	$DOCKER_EXE run --rm -i consul timeout -t 10 consul agent -dev -client 0.0.0.0 | grep "0.0.0.0"
+}
+
+@test "run a gcc container" {
+	$DOCKER_EXE run --rm -i gcc bash -c "echo -e '#include<stdio.h>\nint main (void)\n{printf(\"Hello\");return 0;}' > demo.c && gcc demo.c -o demo && ./demo"
+}
+
+@test "run a crate container" {
+	$DOCKER_EXE run --rm -i -e CRATE_HEAP_SIZE=1g crate timeout -t 10 crate -h | grep "starts"
+}
+
+@test "check the resolv.conf in a crux container" {
+	$DOCKER_EXE run --rm -i crux bash -c "cat /etc/resolv.conf | grep "nameserver""
+}
+
+@test "find the timestamp help in a gazebo container" {
+	$DOCKER_EXE run --rm -i gazebo gz log --help
+}
+
+@test "go env in a golang container" {
+	$DOCKER_EXE run --rm -i golang bash -c "golang go env | grep "GOARCH""
```

Comment:
The quoting on `GOARCH` looks wrong? You can drop the quotes around `GOARCH` itself anyway as it's a static string.